src/protocol.rs

```diff
@@ -367,8 +367,9 @@ impl ParticipantIdentifier {
     }
 }
 
-/// The SharedContext contains fixed known parameters accross the entire
-/// protocol. It does not however contain the entire protocol context.
+// The SharedContext contains fixed known parameters accross the entire
+// protocol. It does not however contain the entire protocol context.
```

Comment:
nit: Does this build when you use `///`? If so, please use that instead.

Also, even if it won't let you link to shared context, please put it in backticks.